src/nextjournal/clerk.clj

```diff
@@ -306,6 +306,10 @@
     `(clerk/with-viewer v/examples-viewer
        (mapv (fn [form# val#] {:form form# :val val#}) ~(mapv (fn [x#] `'~x#) body) ~(vec body)))))
 
+(defmacro card
+  "Evaluates and inspects `body` in SCI context for cljs consumption."
+  [body]
+  `(clerk/with-viewer v/card-viewer '~body))
```

Comment:
I'd rather not have this in the public api. Can you inline it in the notebook for now, even if it means we need to manually evaluate it?